### ISSUE TITLE
NumberFormatException thrown to the caller

### ISSUE THREAD

**Issue by luchen01** [nulab/zxcvbn4j#78]
Hi team, 

I have found that calling `zxcvbn.measure(password)` throws a `NumberFormatException` for some passwords because `len == 0` after the trailing white spaces are trimmed in `WipeableString.java`. I have no information about the password value which led to this case. But I have tested with string of empty spaces or passwords like `2020   9015 `, neither of them threw the exception.

This is the stacktrace:
```
java.lang.NumberFormatException
	com.nulabinc.zxcvbn.WipeableString.parseInt(WipeableString.java:214)
	com.nulabinc.zxcvbn.WipeableString.parseInt(WipeableString.java:154)
	com.nulabinc.zxcvbn.matchers.DateMatcher.execute(DateMatcher.java:43)
	com.nulabinc.zxcvbn.matchers.OmnibusMatcher.execute(OmnibusMatcher.java:31)
	com.nulabinc.zxcvbn.Matching.omnimatch(Matching.java:51)
	com.nulabinc.zxcvbn.Zxcvbn.measure(Zxcvbn.java:33)
	com.nulabinc.zxcvbn.Zxcvbn.measure(Zxcvbn.java:15)
```

One suggestion is to expose Invalid Argument as part of a declared exception or at least document that this exception can be thrown. Or alternatively swallowing the exception may be a better approach here.

Thanks!

**Comment by vvatanabe** [comment]
@luchen01 
Thanks for the report! I will fix it with an approach that swallows exceptions.

**Comment by vvatanabe** [comment]
@luchen01 I fixed and released version 1.3.6
